homeassistant/components/swiss_public_transport/types.py

```diff
@@ -0,0 +1,18 @@
+"""Types for the swiss_public_transport integration."""
+
+from dataclasses import dataclass
+
+from homeassistant import config_entries
+
+from .coordinator import SwissPublicTransportDataUpdateCoordinator
+
+type SwissPublicTransportConfigEntry = config_entries.ConfigEntry[
+    SwissPublicTransportRuntimeData
+]
+
+
+@dataclass
+class SwissPublicTransportRuntimeData:
+    """Runtime information for swiss public transport."""
+
+    coordinator: SwissPublicTransportDataUpdateCoordinator
```

Comment:
```suggestion
```
No need to have a dataclass here, we can just attach the coordinator directly to `runtime_data`